build.sbt

```diff
@@ -17,7 +17,7 @@ libraryDependencies ++= Seq(
   "org.metafacture" % "metafacture-flux" % "5.7.0-rc1",
   "org.metafacture" % "metafacture-triples" % "5.7.0-rc1",
   "org.metafacture" % "metafacture-formatting" % "5.7.0-rc1",
-  "org.metafacture" % "metafix" % "0.6.0-rc3",
+  "org.metafacture" % "metafix" % "0.6.0-SNAPSHOT",
```

Comment:
This causes the build to fail.

@TobiasNx Did you need something specific in the SNAPSHOT, or was this just to make it work for you locally?

Reply:
nope, it was just to make it work locally. The new release should be good.

Reply:
Then I'll revert to `0.6.0-rc3` for now, which is configured in the GitHub CI action.